db/seeds.rb

```diff
@@ -449,3 +451,55 @@
 user_journey_content_2.save!
 
 puts "Finished!"
+
+
+# Start creating fake user at large scale
+
+
+# p "create 20 users"
+
+
+# 20.times do |_index|
+#   User.create!(name: Faker::Name.name,
+#               email: Faker::Internet.email,
+#               password: "123456"
+#                  )
+# end
+
+
+# #  get all users
+
+# users = User.all
+
+# # get all journeys
+
+# journeys = Journey.all
+# journeys_number = Journey.count
+# # each user create a random numbers of journeys 
+
+
+# users.each do |user|
+#   # each user pick up a random number between one to the max number of journey 
+#   random_numbers_of_journey = rand(1..journeys_number)
+#   # p random_numbers_of_journey
+#   journeys_pickup = journeys.sample(random_numbers_of_journey)
+  
+#   journeys_pickup.each do | journey_pickup|
+    
+#     user_journey_created = UserJourney.create!(
+#       journey: journey_pickup,
+#       user: user,
+#       completed: false
+#     )
+      # reset here tomorrow nigth 
+#     p journey_pickup.contents
+#   end
+
+# end
+
+# user_journey_1 = UserJourney.new(
```

Comment:
Already done at lines 489-496 in a more systematic way

Reply:
ca c etait mon copier coller pour me rappeler des params